solr/core/src/java/org/apache/solr/search/SolrIndexSearcher.java

```diff
@@ -706,9 +707,12 @@ public <K, V> boolean regenerateItem(
     }
   }
 
-  public QueryResult search(QueryResult qr, QueryCommand cmd) throws IOException {
-    getDocListC(qr, cmd);
-    return qr;
+  public void search(QueryResult qr, QueryCommand cmd) throws IOException {
+    try {
+      getDocListC(qr, cmd);
+    } catch (FuzzyTermsEnum.FuzzyTermsException e) { // unsure where best to catch this; shrug
```

Comment:
I moved the FuzzyTermsException catch block from QueryComponent to here.  I wanted it in a deeper place that would be caught by multiple inward code paths instead of just QC.  It still feels it ought to be deeper but I didn't look for a better spot.  CombineQuery calls into this.

Reply:
It seems all right to me, but I don't have a strong opinion